### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Chike1990**
In this Milestone, I set up the structure for Leaderboard page using HTML, a little CSS, JS (ES6 import and export modules), gitflow, and webpack.

### CODE REVIEW

**Review by henatan99 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

# Status: Invalid :x: 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

- Your `.gitignore` filename has a typo error. As a result, all the node_modues are being committed. Please fix the filename error and make sure node_modules files are not committed. It is not possible to review the PR. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready. :pray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------------------------------------------------------------------------------------------------
Invalid Code Review Request ### does not count into the code reviews limit.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "LeaderBoard_Project",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (carlos-ssh) [inline package.json]: Your package.json it's missing the line where you start the server:
`  "start": "webpack --config webpack.config.js --watch" `

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Chike1990 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**